server/src/main/java/com/lunchtime/controllers/RestaurantController.java

```diff
@@ -66,6 +66,16 @@ public RestaurantController(RestaurantService restaurantService) {
             .body(restaurantService.getRestaurantPage(pageable));
     }
 
+    @GetMapping("/userId")
+    public ResponseEntity<Page<Restaurant>> getRestaurantPageByUserId(
+        Long userId, Pageable pageable) {
+        Page<Restaurant> restaurantPage = restaurantService.getRestaurantPageByUserId(userId, pageable);
+        if (restaurantPage == null) {
+            return ResponseEntity.notFound().build();
+        }
+        return ResponseEntity.ok().body(restaurantPage);
+    }
+
```

Comment:
I think this method's name should be `getRestaurant**s**ByUserId`, without `Page`, as well as `getRestaurant**s**ByUserId`

Reply:
without page, you won't no how many rests will be returned (all of them or some size). 
Second, this method returns not only list of rests, but page (it has many other fields for page like size, count, number, sort ,ets)